nncf/quantization/layers.py

```diff
@@ -236,6 +248,21 @@ def run_export_quantization(self, x: torch.Tensor):
                                                       input_high)
         if self._export_mode == QuantizerExportMode.ONNX_QUANTIZE_DEQUANTIZE_PAIRS:
             x, y_scale, y_zero_point, per_channel_idx = self._prepare_qdq_export_quantization(x)
+
+            if self.per_channel:
+                if get_torch_version_tuple() < (1, 8, 0):
```

Comment:
I think the inner `if` could be replaced with `and` for better readability.

Reply:
We still have to handle the implicit "else" branch of that inner if by doing `raise RuntimeError...`, and that "else" branch should only be called for specific PT versions.